examples/layers/widgets/spinning_mixed.dart

```diff
@@ -102,6 +102,15 @@ void main() {
   transformBox = RenderTransform(child: flexRoot, transform: Matrix4.identity(), alignment: Alignment.center);
   final RenderPadding root = RenderPadding(padding: const EdgeInsets.all(80.0), child: transformBox);
 
-  binding.renderView.child = root;
+  // TODO(goderbauer): Create a window if embedder doesn't provide an implicit view to draw into.
```

Comment:
Would it make sense to also create a `ViewWidgetFlutterBinding` abstraction?

Reply:
Since this is only used in one place I opted for not creating the abstraction. It is possible to do so, though.